docs/current/api/975146-concepts.md

```diff
@@ -0,0 +1,126 @@
+---
+slug: /api/concepts
+---
+
+# Key Concepts
```

Comment:
I struggled a bit with how best to structure this tutorial. I decided finally on this concepts-based approach, as it seems to fit better for an API than the step-wise sequential walkthrough we have for the SDKs. I'd like to have feedback on whether this approach is OK or if I should consider a different structure altogether.